src/virtual-source-buffer.js

```diff
@@ -348,6 +352,10 @@ export default class VirtualSourceBuffer extends videojs.EventTarget {
       return segmentObj;
     }, sortedSegments);
 
+    if (sortedSegments.audio.info) {
+      this.mediaSource_.changeInfo_('audio', sortedSegments.audio.info);
+    }
```

Comment:
We abort in hls by switching off of this sourebuffer but only in Firefox (so it never gets appended to an active sourebuffer) at least that is what I saw